docs/local-dev.md

```diff
@@ -8,26 +8,26 @@ Run `npm install` after cloning this repository.
 
 ### Client-only mode
 
-If you're only working on client code and don't need to use/test any of the server functionality described below. You can use `npm run start-and-serve` to write development versions of the Iodide client-side app resources to `dev/` and to serve the files in that folder at `http://localhost:8000`. You can open `http://localhost:8000/iodide.dev.html` in your browser to get started with a blank notebook, or open `http://localhost:8000` to see the list of files saved in `dev/` (in case you have exported other test notebooks in that folder)
+If you're only working on client code and don't need to use/test any of the server functionality described below,
+you can skip setting up a full docker environment and get up and running quickly. Just run
+`npm run simple-serve` after `npm install`: this should start up a web server which will provide a basic
+version of the editing environment you can access at http://localhost:8080/ (note this is distinct from the Django-based
+environment, which uses port 8000).
 
-The command `npm run start-and-serve` runs in watch mode, so changes to files will be detected and bundled into `dev/` automatically, but you will need to refresh the page in your browser manually to see the changes -- we have disabled "hot reloading" because automatically refreshing the browser would cause any active notebooks to lose their evaluation state.
+The command runs in watch mode, so changes to files will be detected and bundled automatically, but you will need to refresh the page in your browser manually to see the changes -- we have disabled "hot reloading" because automatically refreshing the browser would cause any active notebooks to lose their evaluation state.
 
-If you require verbose Redux logging, you can use the command `npm run start-and-serve -- reduxVerbose`
-
-#### Exporting from client-only dev mode
-
-In this mode, resource paths are set to be relative to the `dev/` directory. Thus, if you export a bundled notebook from a dev notebook, you need to be sure save the exported HTML file in the `dev/` folder for the relative paths to correctly resolve the required js, css, and font files (and if you want to share a notebook that you created in a dev environment, you'll need to update the paths to point to the web-accessible resources at `iodide.io` and `iodide.app`).
+If you require verbose Redux logging, you can use the command `npm run simple-serve -- reduxVerbose`
```

Comment:
oops, `--reduxVerbose` doesn't work anymore, one needs to set the environment var `REDUX_LOGGING` to `VERBOSE` or `VERY_VERBOSE`-- e.g.:

`REDUX_LOGGING=VERBOSE npm run start`
or
`REDUX_LOGGING=VERBOSE npm run simple-serve`

i think i need to update the docs for this... :-(

Reply:
It seems like this is the only place where `--reduxVerbose` is mentioned, so I think we can just change it here and be done with it.